components/phpunit_bridge.rst

```diff
@@ -0,0 +1,114 @@
+.. index::
+   single: PHPUnitBridge
+   single: Components; PHPUnitBridge
+
+The PHPUnit Bridge Component
+=============================
+
+    The PHPUnit Bridge component provides utilities to report legacy tests and
+    usage of deprecated code.
+
+It comes with the following features:
+
+* Forces the application to use the default language locale for output (set the
+  locale to ``C``)
```

Comment:
What about changing this to: "Forces the tests to use a consistent locale (it's set to ``C``)"